code/modules/nifsoft/nif.dm

```diff
@@ -570,7 +570,7 @@ You can also set the stat of a NIF to NIF_TEMPFAIL without any issues to disable
 	name = "bioadaptive NIF"
 	desc = "A NIF that goes out of it's way to accomidate strange body types. \
 	Will function in species where it normally wouldn't."
-	durability = 25
+	durability = 100 //Vorestation edit, why are we nerfing non-normal organics. fun should be allowed. 25 changed to 100
```

Comment:
Huh, I thought this was a Vorestation only item. The whole Nifsoft module is a Virgo creation, innit? Won't really need the comment in that case.

Reply:
That is correct.

Reply:
Also, I just noticed the desc has a spelling error: `it's way` rather than `its way`

Reply:
Can we NOT have passive-aggressive bullshit left in comments?

Reply:
Fixed